community_server/src/Controller/AppRequestsController.php

```diff
@@ -156,9 +169,6 @@ public function sendCoins()
         if($required_fields !== true) {
             return $this->returnJson($required_fields);
         }
-        if(!isset($params['memo']) || strlen($params['memo']) < 5 || strlen($params['memo']) > 150) {
-            return $this->returnJson(['state' => 'error', 'msg' => 'memo is not set or not in expected range [5;150]']);
-        }
```

Comment:
What happend to those restrictions?

Reply:
The public key is the one identifier which works across communities.
And I have added functions to get the public key from users from another community if you know the community server url and the username, like in your example.

Reply:
But in my plan Login-Server is able to get for the username the correct public key from blockchain of the target community. 
We will see if that is also possible with using Iota.

Reply:
Oh, you are right I have make mistake there. 
They moved to parseParameterForCreateTransaction
but I have forget the strlen destrictions.